containers/roles/foreman/tasks/hacks.yml

```diff
@@ -2,11 +2,11 @@
 - name: 'Add Katello verify SSL'
```

Comment:
It'd be nice if this branch was rebased and submitted as a PR or get the proper certificates into the right servers.

Reply:
Will take this